pages/operations/managing-applications/bounce-application.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Bounce an Application
+description: How to redeploy an application for new configuration or troubleshooting.
```

Comment:
Is it actually redeploying or just restarting?

Reply:
This is a really good question. It redeploys the Helm chart, which is different from redeploying the apps. I've clarified this now!